app/models/user.rb

```diff
@@ -1,2 +1,5 @@
 class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id', dependent: :destroy
+  has_many :comments, foreign_key: 'author_id', dependent: :destroy
+  has_many :likes, foreign_key: 'author_id', dependent: :destroy
 end
```

Comment:
- [ ] Great job so far. please, according to the project requirements, User model has method that returns 3 most recent posts. eg
```ruby
def recent_post
  posts.order('created_at Desc').limit(3)
end
```